oceanmodes/baroclinic.py

```diff
@@ -165,7 +168,240 @@ def _neutral_modes_from_N2_profile_raw(z, N2, f0, depth=None, **kwargs):
     j = np.argsort(w)[::-1]
     w = w[j]
     v = v[:,j]
+    
+    #########
+    # w = - 1/(Rd^2 * f0^2)
+    #########
+    Rd = (-w)**-0.5 / f0
+
+    return zf, Rd, v
+
+def instability_analysis_from_N2_profile(z, N2, f0, beta, Nx, Ny, dx, dy, ubar, vbar, etax, etay, sort='LI', num=4, depth=None, **kwargs):
```

Comment:
This line is too long. According to the [python style guide](https://www.python.org/dev/peps/pep-0008/), lines should not be longer than 80 characters.